### PR TITLE
fixed selectors

### PR DESCRIPTION
**Pull request by kapoliub**

### CODE REVIEW

**Review by moran711 — COMMENTED** [review]
Please change naming of all selectors due to style guide

File: src/hooks/product/use-product-handlers.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { useState, useMemo, useCallback } from 'react';
 import { useSelector } from 'react-redux';
 import { config } from '../../configs';
-import {
-  AdditionsSelector,
-  OptionsSelector
-} from '../../redux/selectors/use-product-handlers.selectors';
 import useProductSpecies from './use-product-species';
 
+const AdditionsSelector = ({ additions }) => additions.length;
```
Comment (moran711) [inline src/hooks/product/use-product-handlers.js:6]: It's make sense to rename it to `selectAdditionsLength `

File: src/hooks/product/use-product-handlers.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+
+const OptionsSelector = ({ options }) => options.find(AdditionsSelector);
```
Comment (moran711) [inline src/hooks/product/use-product-handlers.js:8]: It's make sense to rename it to `selectOptionsLength`

File: src/hooks/product/use-product-species.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { useMemo } from 'react';
 import { useSelector } from 'react-redux';
-import { ColorsSelector } from '../../redux/selectors/use-product-species.selectors';
+
+const ColorsSelector = ({ colors }) => colors[0].simpleName[0].value;
```
Comment (moran711) [inline src/hooks/product/use-product-species.js:4]: same

File: src/hooks/product/use-product-validation.js [hunk 4]
```diff
@@ -5,7 +5,8 @@ import { useSelector } from 'react-redux';
 
 import { config } from '../../configs';
 import { productsTranslations } from '../../translations/product.translations';
-import { NameSelector } from '../../redux/selectors/use-product-validation.selectors';
+
+const NameSelector = ({ name }) => [name, Yup.string().required()];
```
Comment (moran711) [inline src/hooks/product/use-product-validation.js:9]: same

File: src/hooks/product/use-product-handlers.js (hunk 1 above)
Comment (kapoliub) [reply src/hooks/product/use-product-handlers.js]: fixed

File: src/hooks/product/use-product-species.js (hunk 3 above)
Comment (kapoliub) [reply src/hooks/product/use-product-species.js]: fixed

File: src/hooks/product/use-product-handlers.js (hunk 2 above)
Comment (kapoliub) [reply src/hooks/product/use-product-handlers.js]: fixed

File: src/hooks/product/use-product-validation.js (hunk 4 above)
Comment (kapoliub) [reply src/hooks/product/use-product-validation.js]: fixed

**Review by moran711 — APPROVED** [review]

**Review by nadiyafomenko — APPROVED** [review]

**Review by VitalikVoicix — APPROVED** [review]

**Pull request merged**